modules/oe_webtools_maps/tests/src/Functional/ConfigurationTest.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_webtools_maps\Functional;
+
+use Drupal\Core\Entity\Entity\EntityFormDisplay;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Tests\oe_webtools\Traits\ApplicationJsonAssertTrait;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\node\Entity\Node;
+
+/**
+ * Tests that the configured settings are correctly reflected in the page.
+ */
+class ConfigurationTest extends BrowserTestBase {
+
+  use ApplicationJsonAssertTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'geofield',
+    'node',
+    'oe_webtools_maps',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    $this->drupalCreateContentType([
+      'type' => 'test',
+    ]);
+    $field_storage = FieldStorageConfig::create([
+      'field_name' => 'geofield_field',
+      'entity_type' => 'node',
+      'type' => 'geofield',
+      'settings' => [
+        'backend' => 'geofield_backend_default',
+      ],
+    ]);
+    $field_storage->save();
+
+    FieldConfig::create([
+      'field_storage' => $field_storage,
+      'bundle' => 'test',
+      'settings' => [
+        'backend' => 'geofield_backend_default',
+      ],
+    ])->save();
+
+    /** @var \Drupal\Core\Entity\Display\EntityViewDisplayInterface $display */
+    $display = \Drupal::entityTypeManager()
+      ->getStorage('entity_view_display')
+      ->load('node.test.default');
+
+    $display->setComponent('geofield_field', [
+      'label' => 'above',
+      'type' => 'oe_webtools_maps_map',
+    ]);
+    $display->save();
+
+    EntityFormDisplay::load('node.test.default')
+      ->setComponent($field_storage->getName(), [
+        'type' => 'geofield_latlon',
+      ])
+      ->save();
+
+    $this->drupalLogin($this->drupalCreateUser([], NULL, TRUE));
+  }
+
+  /**
+   * Tests if changing configuration changes the map JSON.
+   */
+  public function testConfigurationChanges(): void {
```

Comment:
As we didn't have at all test coverage for oe_webtools_maps probably it makes sense to extend test coverage to assert field formatter settings. See https://github.com/openeuropa/oe_webtools/pull/258/files#diff-93e9dfce3433d92e9cae1a182d4fb153f8b8169475098c75580da33298e5a187R71